src/components/ChatEntry.js

```diff
@@ -1,22 +1,36 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const time = <TimeStamp 
+                  time={props.timeStamp}>
+                </TimeStamp>
+
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={props.sender==='Vladimir'? 'chat-entry local': 'chat-entry remote' } >
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p className={() =>props.onSetColorCallback}>{props.body}</p>
```

Comment:
It looks like onSetColorCallback declared in App.js takes in an argument `color`, but an argument isn't passed to the method here. I do see in ColorChoice.js that you register onSetColorCallback as an event handler to clicking the heart buttons which is the implementation I'd expect to see. 

I think what's missing from the onSetColorCallback is setting the color as state, currently it just returns color. I'll leave a more specific comment about the method in the App.js file

By the time this component is set to render, we should be able to pass in a value representing a color as a prop (instead of a callback function). I'd expect to see something like
```
<p className={props.selectedColor}>{props.body}</p>